firmware/config/engines/mazda/mazda_miata_1_6.cpp

```diff
@@ -164,22 +164,23 @@ void miataNAcommonEngineSettings() {
 	engineConfiguration->engineChartSize = 200;
 
 	setTable(config->crankingCycleBaseFuel, 27.5);
-	config->crankingFuelCoef[0] = 2.8; // base cranking fuel adjustment coefficient
+	//todo: this are now part of crankingFuelCoef, create a linear/exp function or remove?
+	//config->crankingFuelCoef[0] = 2.8; // base cranking fuel adjustment coefficient
 	config->crankingFuelBins[0] = -20; // temperature in C
-	config->crankingFuelCoef[1] = 2.2;
+	//config->crankingFuelCoef[1] = 2.2;
 	config->crankingFuelBins[1] = -10;
-	config->crankingFuelCoef[2] = 1.8;
+	//config->crankingFuelCoef[2] = 1.8;
 	config->crankingFuelBins[2] = 5;
-	config->crankingFuelCoef[3] = 1.5;
+	//config->crankingFuelCoef[3] = 1.5;
 	config->crankingFuelBins[3] = 30;
 
-	config->crankingFuelCoef[4] = 1.0;
+	//config->crankingFuelCoef[4] = 1.0;
 	config->crankingFuelBins[4] = 35;
-	config->crankingFuelCoef[5] = 1.0;
+	//config->crankingFuelCoef[5] = 1.0;
 	config->crankingFuelBins[5] = 50;
-	config->crankingFuelCoef[6] = 1.0;
+	//config->crankingFuelCoef[6] = 1.0;
 	config->crankingFuelBins[6] = 65;
-	config->crankingFuelCoef[7] = 1.0;
+	//config->crankingFuelCoef[7] = 1.0;
```

Comment:
dead code not nice

it looks like we want to kills some legacy mazda preset values, let's do this in separate preparation PR @FDSoftware ?